### PR TITLE
Merge Tribute page

### PR DESCRIPTION
**Pull request by Snooperton**

### CODE REVIEW

File: Tribute-page HTML code [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <li>Had 14 key principles to offer managers to improve business effectives.</li>
+      <li>Had the "Seven Deadly Diseases" and "Lesser Category of Obstacles".</li>
+      <li>Created the Plan-Do-Study-Act (PDSA) cycle.</li>
+      <li>Was very influential in helping Japan make high-quality products.</li>
+      <li>Has received several rewards such as National Medal of Techonology in 1987.</li>
+      <li>Passed away on 20th of December 1993 in Washington DC.
+    </ul>
+  </div>
+  <div id="external-link">
+    <p>You can read more about W.Edwards Deming on <a id="tribute-link" target="_blank" href="https://en.wikipedia.org/wiki/W._Edwards_Deming">Wikipedia</a>.</p>
```
Comment (nicholasmueller) [inline Tribute-page HTML code]: you should nest the <a> tag so its more readable

```
<p>
  something
  <a>
     wikipedia
  </a>
</p>

File: Tribute-page CSS code [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#main {
```
Comment (nicholasmueller) [inline Tribute-page CSS code:1]: noice.. try with classes instead of ids so you can reuse some css

**Review by nicholasmueller — APPROVED** [review]
Nice work!!!!!!!!!!!!!

File: Tribute-page HTML code (hunk 1 above)
Comment (Snooperton) [reply Tribute-page HTML code]: ah cool, defs will use in the future.

File: Tribute-page CSS code (hunk 2 above)
Comment (Snooperton) [reply Tribute-page CSS code:1]: yeah, the project had to be done with 'id' specifically to pass it. had like 10 tests to pass

File: Tribute-page HTML code [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      <li>Had the "Seven Deadly Diseases" and "Lesser Category of Obstacles".</li>
+      <li>Created the Plan-Do-Study-Act (PDSA) cycle.</li>
+      <li>Was very influential in helping Japan make high-quality products.</li>
+      <li>Has received several rewards such as National Medal of Techonology in 1987.</li>
+      <li>Passed away on 20th of December 1993 in Washington DC.
+    </ul>
+  </div>
+  <div id="external-link">
+    <p>You can read more about W.Edwards Deming on 
+    <a id="tribute-link" target="_blank" href="https://en.wikipedia.org/wiki/W._Edwards_Deming">Wikipedia</a>.
```
Comment (Snooperton) [inline Tribute-page HTML code:29]: is this what u meant by nesting it

**Pull request merged**